packages/odata-annotation-core/package.json

```diff
@@ -30,7 +30,8 @@
         "watch": "tsc  --build ./ -watch"
     },
     "dependencies": {
-        "@sap-ux/odata-annotation-core-types": "workspace:*"
+        "@sap-ux/odata-annotation-core-types": "workspace:*",
+        "@sap-ux/metadata": "workspace:*"
```

Comment:
Since metadata service is only used in tests, it should be a dev dependency.

Reply:
moved